packages/amplify-provider-awscloudformation/src/admin-modelgen.ts

```diff
@@ -1,13 +1,11 @@
 import {
-  $TSAny, $TSContext, AmplifyError, AmplifyFault, pathManager, stateManager,
+  $TSAny, $TSContext, pathManager, stateManager,
 } from 'amplify-cli-core';
-import { printer } from 'amplify-prompts';
-import AWS from 'aws-sdk';
-import { PromiseResult } from 'aws-sdk/lib/request';
+import * as fs from 'fs-extra';
 import { isDataStoreEnabled } from 'graphql-transformer-core';
-import ora from 'ora';
+import _ from 'lodash';
 import * as path from 'path';
-import { AmplifyBackend } from './aws-utils/aws-amplify-backend';
```

Comment:
Probably can delete this file since this is the only dependency?

Reply:
done